react/components/EXPERIMENTAL_Table/hooks/useTableBulkActions.tsx

```diff
@@ -176,19 +179,22 @@ function reducer(state: BulkState, action: Action) {
       }
     }
     case ActionType.SelectRow: {
+      const {
+        rowToToggle: { row, unicityKey },
+      } = action
       return state.selectedRows.some(
-        selectedRow => selectedRow.id === action.row.id
+        selectedRow => selectedRow[unicityKey] === row[unicityKey]
       )
         ? {
             ...state,
             selectedRows: state.selectedRows.filter(
-              row => row.id !== action.row.id
+              r => r[unicityKey] !== row[unicityKey]
```

Comment:
Nitpick: keep the `row` as variable name instead of something less readable like `r`.

Reply:
Oops, done. Thanks